all2vec/__init__.py

```diff
@@ -270,3 +271,53 @@ def load(cls, folder):
                     'but was not loaded'.format(enttype['entity_type'])
                 )
         return unpickled_class
+
+    @classmethod
+    def load_remote_subset(cls, nfactor, entities, sparkfiles):
+        """Load subset of ann objects from hdfs or s3.
+
+        Requires SparkFiles (imported from pyspark),
+        must have sc.addPyFile(path + file) prior to SparkFiles.get(file).
+        entities is a list of annoy indexes to load.
+        """
+        with open(sparkfiles.get("object.pickle")) as f:
```

Comment:
Updated: https://github.com/iheartradio/all2vec/pull/6/files#diff-b160d7b80f8d507ee4e0536285c95431R282 and added a test https://github.com/iheartradio/all2vec/pull/6/files#diff-6492451bd05313d54e9ab1b5d2f97307R77